tests/test_bundle_get_data.py

```diff
@@ -53,7 +53,7 @@ def test_get_all_bundles_list(self, params):
             output = get_all_bundles_list(**params)
             self.assertTrue(isinstance(output, list))
             self.assertTrue(isinstance(output[0], tuple))
-            self.assertTrue(len(output[0]) == 2)
+            self.assertEqual(output[0], 2)
```

Comment:
```suggestion
            self.assertEqual(len(output[0]), 2)
```

Looks like len should be added here?

Reply:
You're absolutely right, and sorry for making this mistake.